### ISSUE TITLE
revert, undo conversion?

### ISSUE THREAD

**Issue by 1isten** [ivanceras/svgbob#45]
Can I revert an svg file(converted by svgbob) back to the original ascii text?

**Comment by ivanceras** [comment]
No, it requires a very complex algorithm. A quick hack however is to embed the original text in the svg and reverting back to original ascii is merely just extracting the embedded text. It has no practical purpose since you already have a copy of this original text. If you are thinking of editing the svg and the ascii text change as well, then it is not possible.

**Comment by 1isten** [comment]
What's the right way to embed the original text in the svg?

(By the way the purpose here is I was using ascii text in markdown to write some posts, but I found svg be more responsive so I converted the ascii code to svg images, thanks for your great tool. But in case someday I need to edit/update the some images with a small change, but I don't keep the original ascii text copy, then I have to make re create the ascii text again to get my new svg...)
